bin/morgue.js

```diff
@@ -2872,6 +2872,16 @@ function coronerGet(argv, config) {
       var fname = getFname(hr, out.path, argv.outdir, objects.length, oid, params.resource);
       success++;
       if (fname) {
+        if (hr.headers["content-encoding"] === "gzip" && params.resource === "json.gz") {
```

Comment:
Can we add txt.gz here too? Would be useful.

Reply:
Done, and updated.